bears/vcs/git/GitCommitBear.py

```diff
@@ -305,25 +315,24 @@ def check_issue_reference(self, body,
             return
 
         host = self.get_host_from_remotes()
-        if host not in self.SUPPORTED_HOST_KEYWORD_REGEX:
+        if (host not in self.SUPPORTED_HOST_KEYWORD_REGEX or
+                host not in self.ISSUE_INFO):
             return
 
+        if body_close_issue_full_url:
+            self.ISSUE_INFO['type'] = 'full issue'
```

Comment:
dont modify the *class* constant `self.ISSUE_INFO`.

It stops being a constant.

use an instance variable.